spec/lib/extensions/ar_taggable_spec.rb

```diff
@@ -16,6 +16,24 @@
     @vm4.tag_add("bos phi blt")
   end
 
+  describe '#writable_classification_tags' do
+    let(:parent_classification) { FactoryGirl.create(:classification, :description => "Environment", :name => "environment", :read_only => false) }
+    let(:classification)        { FactoryGirl.create(:classification, :name => "prod", :description => "Production", :parent => parent_classification, :read_only => true) }
+
+    before do
+      classification.assign_entry_to(@vm1)
+    end
+
+    it "returns only tags as they would be entered in UI by user(edit tags screen)" do
+      expect(@vm1.tags.count).to eq(4)
+      expect(@vm1.writable_classification_tags.count).to eq(1)
+      expect(@vm1.writable_classification_tags.first.name).to eq('/managed/environment/prod')
+
```

Comment:
Could you add an expectation to check if the returned object is an actual `tag`? Also to test if the joined description I asked for is available.

Reply:
done